cmd/init_tezos.go

```diff
@@ -60,16 +60,15 @@ var initTezosCmd = &cobra.Command{
 }
 
 func validateTezosFlags() error {
-	if initOptions.RemoteNodeURL == "" {
-		return fmt.Errorf("you must provide 'remote-node-url' flag as local node mode is not supported")
+	if initOptions.ExtraConnectorConfigPath == "" {
+		return fmt.Errorf("you must provide 'connector-config' flag with path to 'tezosconnect.yml' config file")
```

Comment:
Hey @alex-semenyuk, thanks for the PR!
'tezosconnect.yml' is not a required file, it overrides the default config of the service ('tezossigner.yaml').
'remote-node-url' option is required to initialise the default config, so it has higher priority over the 'tezosconnect.yml'.

Reply:
@denisandreenko thanks for clarification then we can close it but I raised [one more](https://github.com/hyperledger/firefly/pull/1437) to improve documentation on this.